lib/rules/no-unused-prop-types.js

```diff
@@ -949,7 +949,7 @@ module.exports = {
     }
 
     function handleSetStateUpdater(node) {
-      if (!node.params || !node.params.length || !inSetStateUpdater()) {
+      if (!node.params || !node.params.length || node.params.length < 2 || !inSetStateUpdater()) {
```

Comment:
`!node.params.length` is `node.params.length === 0`, so this is now redundant with the "< 2" comparison. let's remove the former.